tests/bot_tests/SOA_test.py

```diff
@@ -62,7 +62,7 @@ def test_play(self):
 
     @testing.gen_test
     def test_send_message(self):
-        hostname = "shade.tacc.utexas.edu"
+        hostname = "shade-dev.tacc.utexas.edu"
```

Comment:
Might make sense to add flag for dev vs main tacc down the road.  Fine for this and not sure of significance between the two.

Reply:
I have no clue. This is the only spot it's referenced in the main code, and changing it didn't affect anything.